skyagi-web/src/hooks.server.ts

```diff
@@ -1,42 +1,42 @@
 // src/hooks.server.ts
 import {
-  PUBLIC_SUPABASE_URL,
-  PUBLIC_SUPABASE_ANON_KEY
+    PUBLIC_SUPABASE_URL,
+    PUBLIC_SUPABASE_ANON_KEY
 } from '$env/static/public';
 import { createSupabaseServerClient } from '@supabase/auth-helpers-sveltekit';
-import type { Handle } from '@sveltejs/kit';
+import { redirect, type Handle } from '@sveltejs/kit';
 
 export const handle: Handle = async ({ event, resolve }) => {
-  event.locals.supabase = createSupabaseServerClient({
-    supabaseUrl: PUBLIC_SUPABASE_URL,
-    supabaseKey: PUBLIC_SUPABASE_ANON_KEY,
-    event
-  });
+    event.locals.supabase = createSupabaseServerClient({
+        supabaseUrl: PUBLIC_SUPABASE_URL,
+        supabaseKey: PUBLIC_SUPABASE_ANON_KEY,
+        event
+    });
 
-  /**
-   * A convenience helper so we can just call await getSession() instead const { data: { session } } = await supabase.auth.getSession()
-   */
-  event.locals.getSession = async () => {
-    const {
-      data: { session }
-    } = await event.locals.supabase.auth.getSession();
-    return session;
-  };
-
-  // Get the cookies from the request
-	const { cookies } = event;
-
-	// Get the user token from the cookie
-	const authToken = cookies.get('sb-auth-token');
```

Comment:
we don't need the authToken to authorize supabase?

Reply:
Instead using `const session = await event.locals.getSession();` to get whether user is logged in or not, it will use the `sb-auth-token` cookie behind the scenes.